testing/mu/APKBUILD

```
@@ -1,40 +1,41 @@
 # Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
 # Contributor: stef <l0ls0fo2i@ctrlc.hu>
+# Contributor: Eivind Uggedal <eu@eju.no>
 # Maintainer:
 pkgname=mu
-pkgver=1.0
+pkgver=1.2
+_pkgver=1.2.0
 pkgrel=0
 pkgdesc="A collection of utilities for indexing and searching Maildirs"
 arch="x86 x86_64 ppc64le"
 url="http://www.djcbsoftware.nl/code/mu"
 license="GPL-3.0"
 depends="sqlite"
-makedepends="xapian-core-dev gmime-dev sqlite-dev autoconf automake libtool"
-install=""
-subpackages="$pkgname-doc"
-source="$pkgname-$pkgver.tar.gz::https://github.com/djcb/mu/archive/v$pkgver.tar.gz
-	0001-Support-gmime-3.0.patch
-	"
+makedepends="xapian-core-dev gmime-dev sqlite-dev emacs autoconf automake
+	libtool"
+subpackages="$pkgname-doc mu4e"
+source="https://github.com/djcb/mu/releases/download/$pkgver/mu-$_pkgver.tar.xz"
```

```suggestion
source="https://github.com/djcb/mu/releases/download/${pkgver:0:-2}/mu-$pkgver.tar.xz"
```
Use the default pkgver variable and simply strip the last two chars.

---

That won't work since the upstream version is `1.2`, not `1.2.0`.